app/src/test/java/com/github/swent/echo/compose/association/AssociationMainScreenTest.kt

```diff
@@ -0,0 +1,128 @@
+package com.github.swent.echo.compose.association
+
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onNodeWithTag
+import androidx.compose.ui.test.performClick
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.github.swent.echo.data.model.Association
+import com.github.swent.echo.data.model.Event
+import com.github.swent.echo.data.model.EventCreator
+import com.github.swent.echo.data.model.Location
+import com.github.swent.echo.viewmodels.association.AssociationPage
+import java.time.ZonedDateTime
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+enum class TestExpandableLists(val value: String) {
+    FOLLOWED_ASSOCIATIONS("Followed Associations"),
+    MY_ASSOCIATIONS("My Associations")
+}
+
+@RunWith(AndroidJUnit4::class)
+class AssociationMainScreenTest {
+
+    @get:Rule val composeTestRule = createComposeRule()
+
+    private val testAssociations =
+        listOf(
+            Association("id 1", "name 1", "description 1"),
+            Association("id 2", "name 2", "description 2"),
+            Association("id 3", "name 3", "description 3")
+        )
+    private val testEvents =
+        listOf(
+            Event(
+                "event 1",
+                EventCreator.EMPTY,
+                testAssociations[0],
+                "title 1",
+                "",
+                Location.EMPTY,
+                ZonedDateTime.now(),
+                ZonedDateTime.now(),
+                emptySet(),
+                0,
+                0,
+                0
+            ),
+            Event(
+                "event 2",
+                EventCreator.EMPTY,
+                testAssociations[1],
+                "title 2",
+                "",
+                Location.EMPTY,
+                ZonedDateTime.now(),
+                ZonedDateTime.now(),
+                emptySet(),
+                0,
+                0,
+                0
+            ),
+            Event(
+                "event 3",
+                EventCreator.EMPTY,
+                testAssociations[2],
+                "title 3",
+                "",
+                Location.EMPTY,
+                ZonedDateTime.now(),
+                ZonedDateTime.now(),
+                emptySet(),
+                0,
+                0,
+                0
```

Comment:
Using `Event.EMPTY.copy(organizer = ..., title = ..., ...)` instead of instantiating an `Event` from scratch will provide better maintainability to this part of the code (e.g. if there are changes to the `Event` data class).

Reply:
That's true, I have done the change.

Reply:
Thanks!